test/sqllogictest/unsigned_int.slt

```diff
@@ -7,26 +7,13 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0.
 
-# This comment was copied from map.slt
-# The unsigned types have an uncommon OID. If tokio-postres (the driver
-# used
-# by sqllogictest) encounters an OID it doesn't recognize (uint2,
-# uint4, or uint8 in
-# this case), then it queries pg_type (a wrapper around mz_types) for
-# information about it. Our unsigned types currently don't have entries
-# in
-# mz_types, so that query fails and tokio-postgres is unable to execute
-# queries with unsigned types. As a workaround until unsigned types
-# are reflected in pg_type,
-# we just convert everything to `numeric`.
-
-query I
-SELECT 42::uint2::numeric
+query I
+SELECT 42::uint2
 ----
 42
 
 query I
-SELECT 32768::uint2::numeric
```

Comment:
What's the error without the numeric cast? tokio_postgres complains that it doesn't know about that oid?

Reply:
Ah I see this is getting fixed in the other PR. Ok.